File: 1_Two_Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 1; i < nums.length; i++) {
            for (int j = i; j < nums.length; j++) {
                if (target == nums[j] + nums[j - i]) {
                    return new int[]{j, j-i};
                }
            }
        }
        return null;
    }
}

File: 1_Two_Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        m = {}
        for i in range(len(nums)):
            if nums[i] in m:
                return [i, m[nums[i]]]
            m[target - nums[i]] = i
        